### PR TITLE
Possibility to specify the rules to be converted

### PR DESCRIPTION
**Pull request by DenysTsi**
Added possibility to specify the rules to be converted

### CODE REVIEW

File: vl.Core/Domain/Activity/SysmonEventId.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      FileCreateStreamHash = 15,
+      PipeEvent = 17,
+      WmiEvent = 19,
+      DNSQuery = 22,
+      FileDelete,
+      ClipboardChange,
+      ProcessTampering, 
+      FileDeleteDetected
+   }
+}
```
Comment (svnscha) [inline vl.Core/Domain/Activity/SysmonEventId.cs:25]: Please use fixed constants as they don't change.

File: vl.Sysmon.Converter/Program.cs [hunk 2]
```diff
@@ -66,24 +71,78 @@ private static void Main(string[] args)
...
+                     case SysmonEventId.FileCreateTime:
+                     case SysmonEventId.RawAccessRead:
+                     case SysmonEventId.ProcessAccess:
+                     case SysmonEventId.FileCreate:
+                     case SysmonEventId.PipeEvent:
+                     case SysmonEventId.WmiEvent:
+                     case SysmonEventId.DriverLoad:
+                     case SysmonEventId.FileDelete:
+                     case SysmonEventId.ClipboardChange:
+                     case SysmonEventId.ProcessTampering:
```
Comment (svnscha) [inline vl.Sysmon.Converter/Program.cs:133]: How about implementing Process Tampering? ;-)

**Review by svnscha — CHANGES_REQUESTED** [review]

File: vl.Core/Domain/Activity/SysmonEventId.cs (hunk 1 above)
Comment (DenysTsi) [reply vl.Core/Domain/Activity/SysmonEventId.cs:25]: Had also thought about it, is adjusted.

File: vl.Sysmon.Converter/Program.cs (hunk 2 above)
Comment (DenysTsi) [reply vl.Sysmon.Converter/Program.cs:133]: @svnscha in another feature branch, thanks.

**Pull request merged**